app/src/main/java/com/work/restaurant/view/search/itemdetails/presenter/SearchItemDetailsPresenter.kt

```diff
@@ -1,46 +1,29 @@
 package com.work.restaurant.view.search.itemdetails.presenter
 
 import android.util.Log
-import com.work.restaurant.data.repository.fitness.FitnessItemRepository
-import com.work.restaurant.data.repository.fitness.FitnessItemRepositoryCallback
 import com.work.restaurant.data.repository.kakao.KakaoRepository
 import com.work.restaurant.data.repository.kakao.KakaoRepositoryCallback
-import com.work.restaurant.network.model.FitnessCenterItemResponse
-import com.work.restaurant.network.model.kakaoImage.KakaoImageDocuments
 import com.work.restaurant.network.model.kakaoSearch.KakaoSearchDocuments
 
 class SearchItemDetailsPresenter(
     private val searchItemDetailsView: SearchItemDetailsContract.View,
-    private val fitnessItemRepository: FitnessItemRepository,
     private val kakaoRepository: KakaoRepository
 ) :
     SearchItemDetailsContract.Presenter {
-    override fun kakaoItemImage(searchItem: String) {
-
-        kakaoRepository.getKakaoImage(
-            searchItem,
-            object : KakaoRepositoryCallback.KakaoImageCallback {
-                override fun onSuccess(item: List<KakaoImageDocuments>) {
-                    searchItemDetailsView.showKakaoImage(item)
-                }
-
-                override fun onFailure(message: String) {
-                    Log.d("error", message)
-                }
-            })
-
-
-    }
 
     override fun kakaoItemInfoDetail(searchItem: String) {
 
         kakaoRepository.getKakaoItemInfo(
             searchItem,
             object : KakaoRepositoryCallback.KakaoItemInfoCallback {
 
-                override fun onSuccess(item: KakaoSearchDocuments) {
+                override fun onSuccess(item: List<KakaoSearchDocuments>) {
+
+                    val toKakaoSearchModel = item.map{
```

Comment:
```suggestion
                    val kakaoSearchModelList = item.map{
```
map 결과가 list이므로 적절한 네이밍이 필요할 것 같아요.

Reply:
fb49a24 수정하였습니다.